### ISSUE TITLE
Switch out https://github.com/zertosh/eslint-plugin-flow-vars

### ISSUE THREAD

**Issue by hzoo** [babel/babel#4681]
It's deprecated https://github.com/zertosh/eslint-plugin-flow-vars

The functionality provided by `eslint-plugin-flow-vars` has been merge into [eslint-plugin-flowtype](https://github.com/gajus/eslint-plugin-flowtype).

To migrate, [install](https://github.com/gajus/eslint-plugin-flowtype#installation) `eslint-plugin-flowtype@2.4.0` or higher, and update your eslint config like this:

```diff
{
  "parser": "babel-eslint",
  "plugins": [
-    "flow-vars"
+    "flowtype"
  ],
  "rules": {
-    "flow-vars/define-flow-type": 1,
-    "flow-vars/use-flow-type": 1
+    "flowtype/define-flow-type": 1,
+    "flowtype/use-flow-type": 1
  }
}
```

https://github.com/babel/babel/blob/master/package.json#L27

**Comment by hzoo** [comment]
Actually I see we did this in https://github.com/babel/eslint-config-babel/pull/2 but didn't publish

**Comment by danez** [comment]
yeah i wanted to make a PR, going to do it now.

**Comment by hzoo** [comment]
Ok I published a 2.0.0 since otherwise it would break https://github.com/babel/eslint-config-babel/commit/073cafaea7a48e41054a58432e83d484b06294d7
